<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LibWizard</class>
 <widget class="QWizard" name="LibWizard">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Library wizard</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::MacStyle</enum>
  </property>
  <property name="options">
   <set>QWizard::NoDefaultButton</set>
  </property>
  <widget class="QWizardPage" name="modePage">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Application mode</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="spacing">
        <number>12</number>
       </property>
       <item>
        <widget class="QRadioButton" name="mode_lib">
         <property name="text">
          <string>library mode</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="mode_convert">
         <property name="text">
          <string>converter mode</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage1">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Select book's directory:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="Dir"/>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage2">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Select INPX file (optionally):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="inpx"/>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Set library name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lib_name"/>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage_2">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Book's directory:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="r_dir">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>INPX file:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="r_inpx">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Library name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="r_name">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="update_lib">
      <property name="text">
       <string>Update library</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
